mods/seasonal/pokedex.js

```diff
@@ -20,6 +20,10 @@ let BattlePokedex = {
 		inherit: true,
 		baseStats: {hp: 90, atk: 40, def: 100, spa: 120, spd: 240, spe: 100},
 	},
+	flareon: {
+		inherit: true,
+		types: ['Fairy', 'Fire'],
```

Comment:
On second though, since EV's set is using flareon too (as a temp forme) we shouldnt modify it here. Add it as a type in statuses.js